signer/src/network/mod.rs

```diff
@@ -39,7 +38,7 @@ pub trait MessageTransfer: Clone {
     /// Return a stream of the same messages from the
     /// [`MessageTransfer::receive`]. The inner type of each
     /// [`SignerSignal`] should be a [`P2PEvent::MessageReceived`].
-    fn receiver_stream(&self) -> BroadcastStream<SignerSignal>;
```

Comment:
Much happier that this is `Msg` now :pray:

Reply:
Yeah I was very happy. But now https://github.com/stacks-network/sbtc/pull/907/commits/b1d0873c4c169a8b84e4580b223b96716f6a10c4 removed all of that nonsense.